rules/always-return.js

```diff
@@ -139,6 +169,14 @@ module.exports = {
           ignoreLastCallback: {
             type: 'boolean',
           },
+          ignoreAssignmentVariable: {
+            type: 'array',
+            items: {
+              type: 'string',
+              pattern: '^\\w+$',
```

Comment:
```suggestion
              pattern: '^[\\w$]+$',
```

`$` is a fairly common type of identifier part so should be allowed.